hail/python/hail/ggplot/geoms.py

```diff
@@ -484,6 +485,102 @@ def geom_histogram(mapping=aes(), *, min_val=None, max_val=None, bins=None, fill
     return GeomHistogram(mapping, min_val=min_val, max_val=max_val, bins=bins, fill=fill, color=color, alpha=alpha, position=position, size=size)
 
 
+def _max_entropy_cdf(min_x, max_x, x, y, e):
```

Comment:
This function will be hard to maintain by anyone else without significant effort. Please can you split the code up with some functions and/or provide a link to whatever you're basing this implementation off.

Reply:
You're right, of course. When I thought we'd be using this in the workshop, I just wanted to get the change in fast. This is a copy of the same function in the old plotting library, I'll find a better place for this to live so both can use the same function.

This is a completely novel method, so there are no references to link to. I don't know how to explain it without drawing pictures, but I'll do my best.